packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testGetIconIfBelowThreshold.st

```diff
@@ -0,0 +1,12 @@
+tests
+testGetIconIfBelowThreshold
+
+	| model messageTally |
+	
+	model := IPTTreeModelMock new. 
+	messageTally := MessageTallyMock new. 
+
+	messageTally tally: 49.
+	model totalTime: 100.
```

Comment:
This can be merged with the instantiation statements. (See my comment on `testGetIconIfAboveThreshold`)